src/v2/guide/render-function.md

```diff
@@ -6,7 +6,7 @@ order: 303
 
 ## Basics
 
-Vue recommends using templates to build your HTML in the vast majority of cases. There are situations however, where you really need the full programmatic power of JavaScript. That's where you can use the **render function**, a closer-to-the-compiler alternative to templates.
+Vue recommends using templates to build your HTML in the vast majority of cases. There are situations however, where you really need the full programmatic power of JavaScript. That's where you can use the **`render()` function**, a closer-to-the-compiler alternative to templates.
```

Comment:
"Render function" is the right choice of words here, because, well, it's the name of the (type of this) function.